packages/core/src/lib/Canvas.svelte

```diff
@@ -103,15 +103,16 @@
   const { createRenderer } = useRenderer(ctx)
 
   onMount(() => {
-    if (!canvas) return
     createRenderer(canvas, rendererParameters)
     startFrameloop(contexts.ctx, contexts.internalCtx)
     initialized = true
   })
 
   onDestroy(() => {
     contexts.internalCtx.dispose(true)
-    contexts.ctx.renderer?.setAnimationLoop(null)
+
+    // Renderer is marked as optional because it is never defined in SSR
+    contexts.ctx.renderer?.dispose()
```

Comment:
This is still one case where an optional check needs to occur because `onMount` is never called during SSR but `onDestroy` is. Also, `dispose` stops the animation loop internally.

Reply:
Makes sense!